packages/core/android/src/main/kotlin/src/DdSdkReactNativePackage.kt

```diff
@@ -0,0 +1,78 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.reactnative
+
+import com.facebook.react.TurboReactPackage
+import com.facebook.react.bridge.NativeModule
+import com.facebook.react.bridge.ReactApplicationContext
+import com.facebook.react.module.model.ReactModuleInfo
+import com.facebook.react.module.model.ReactModuleInfoProvider
+
+/**
+ * Package of native dd-sdk-reactnative native modules.
+ */
+class DdSdkReactNativePackage : TurboReactPackage() {
```

Comment:
So this is using `TurboReactPackage` which comes from the new architecture, but the class itself sits in the `main` source set, and not in `newarch`? this won't cause problems?

Reply:
This won't cause problems as there is retro compatibility for the old architecture in TurboReactPackage (see the `isTurboModule` variables further down when registering the packages).